tests/test_main_develop.py

```diff
@@ -0,0 +1,111 @@
+'''
+Simple tests for testing functions in main_develop module
+'''
+import os
+import shutil
+from os.path import dirname, join, exists
+
+from conda_build.main_develop import uninstall, write_to_conda_pth
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def sp_dir(request):
+    '''
+    create site-packges/ directory in same place where test is located. This
+    is where tests look conda.pth file. It is a session scoped fixture and
+    it has a finalizer function invoked in the end to remove site-packages/
+    directory
+    '''
+    base_dir = dirname(__file__)
+    sp = join(base_dir, 'site-packages')
+    if exists(sp):
+        shutil.rmtree(sp)
+
+    os.mkdir(sp)
+
+    def cleanup():
+        # session scoped cleanup is called at end of the session
+        shutil.rmtree(sp)
+
+    request.addfinalizer(cleanup)
+
+    return sp
+
+
+@pytest.fixture(scope="function")
+def conda_pth(sp_dir):
+    '''
+    Returns the path to conda.pth - though we don't expect name to change
+    from conda.pth, better to keep this in one place
+
+    Removes 'conda.pth' if it exists so each test starts without a conda.pth
+    file
+    '''
+    pth = join(sp_dir, 'conda.pth')
+    if exists(pth):
+        os.remove(pth)
+
+    return pth
+
+
+# Note: following list is data used for testing - do not change it
+_path_in_dev_mode = ['/Users/jsandhu/Documents/projects/CythonExample',
```

Comment:
Yes! That was the idea - they are simple but useful. The CI should pass these tests.